test/onnx/test_pytorch_onnx_caffe2_opset10.py

```diff
@@ -0,0 +1,15 @@
+import torch
+
```

Comment:
We should have a test, which export some ops which are defined in previous opsets, like opset 9.